darts/timeseries.py

```diff
@@ -2438,94 +2439,97 @@ def with_columns_renamed(
+        """Simple wrapper around :func:`np.ndarray.var()` for stochastic series."""
+        self._assert_stochastic()
+        new_data = self._xa.values.var(axis=2, ddof=1, keepdims=True)
+        new_xa = xr.DataArray(new_data, dims=self._xa.dims, coords=self._xa.coords)
+        return self.__class__(new_xa)
 
-    def std(
-        self, axis=None, skipna=True, level=None, ddof=1, numeric_only=None, **kwargs
-    ) -> float:
-        """Simple wrapper around :func:`pd.DataFrame.std()` for deterministic series."""
-        return self.pd_dataframe(copy=False).std(
-            axis, skipna, level, ddof, numeric_only, **kwargs
-        )
+    def std(self, ddof: int = 1) -> "TimeSeries":
+        """Simple wrapper around :func:`np.ndarray.std()` for stochastic series."""
+        self._assert_stochastic()
+        new_data = self._xa.values.std(axis=2, ddof=1, keepdims=True)
+        new_xa = xr.DataArray(new_data, dims=self._xa.dims, coords=self._xa.coords)
+        return self.__class__(new_xa)
 
-    def skew(
-        self, axis=None, skipna=True, level=None, numeric_only=None, **kwargs
-    ) -> float:
-        """Simple wrapper around :func:`pd.DataFrame.skew()` for deterministic series."""
-        return self.pd_dataframe(copy=False).skew(
-            axis, skipna, level, numeric_only, **kwargs
+    def skew(self, bias: bool = True, nan_policy: str = "propagate") -> "TimeSeries":
+        """Simple wrapper around :func:`scipy.stats.skew()` for stochastic series."""
+        self._assert_stochastic()
+        axis = 2
+        new_data = np.expand_dims(
+            skew(self._xa.values, axis, bias, nan_policy), axis=axis
         )
+        new_xa = xr.DataArray(new_data, dims=self._xa.dims, coords=self._xa.coords)
+        return self.__class__(new_xa)
 
     def kurtosis(
-        self, axis=None, skipna=True, level=None, numeric_only=None, **kwargs
-    ) -> float:
-        """Simple wrapper around :func:`pd.DataFrame.kurtosis()` for deterministic series."""
-        return self.pd_dataframe(copy=False).kurtosis(
-            axis, skipna, level, numeric_only, **kwargs
+        self, fisher: bool = True, bias: bool = True, nan_policy: str = "propagate"
+    ) -> "TimeSeries":
+        """Simple wrapper around :func:`pd.DataFrame.kurtosis()` for stochastic series."""
+        self._assert_stochastic()
+        axis = 2
+        new_data = np.expand_dims(
+            kurtosis(self._xa.values, axis, fisher, bias, nan_policy), axis=axis
         )
+        new_xa = xr.DataArray(new_data, dims=self._xa.dims, coords=self._xa.coords)
+        return self.__class__(new_xa)
 
-    def min(
-        self, axis=None, skipna=True, level=None, numeric_only=None, **kwargs
-    ) -> float:
-        """Simple wrapper around :func:`pd.DataFrame.min()` for deterministic series."""
-        return self.pd_dataframe(copy=False).min(
-            axis, skipna, level, numeric_only, **kwargs
-        )
+    def min(self) -> "TimeSeries":
```

Comment:
this sounds good. Should we provide the same functionality for `mean` and `median` as well? As these are common aggregation function I think it would fit in.

Reply:
Yes, good idea 👍